GOFevaluation/utils.py

```diff
@@ -47,6 +49,106 @@ def equiprobable_histogram(data_sample, reference_sample, n_partitions,
     return n, bin_edges
 
 
+def limit_bin_edges(bin_edges, data_sample, order):
```

Comment:
Maybe call it `get_finite_bin_edges`? This makes it a bit more intuitive to see what this function does. What do you think?